### ISSUE TITLE
Clarify rules surrounding multi-accounting for accessibility to plural systems

### ISSUE THREAD

**Issue by l4ssc** [ppy/osu-wiki#8941]
Hello, we recently found out that we are an OSDD-1 system with quite a few alters. Many of us like playing osu!, but the wording surrounding the rules for multi-accounting are ambiguous for people with experiences like ours:

> **Each player may only have ONE account, ever.** The *first* account you make during account registration is your one and only osu! account, from the moment of its creation and forevermore. This account is YOU. It is not anyone else — not your brother, your mother, your sister, your friend — it is YOU. Don't share your account with anyone else. You don't get to make a new one if you lose access to it. Keep it safe.

Being multiple people inside of a single body, this creates a "damned if you do, damned if you don't" situation where our only option is to not play. We'd really like to be able to enjoy the game without worrying that we might get banned for a condition that is out of our control, though, and request that the wording of this be changed to accommodate systems like us. This would make the rule unambiguous to systems while still preventing everyone it did previously from multi-accounting, so this is only an accessibility improvement to a minor edge case. Something like this would be preferable in our eyes:

> **Each player may only have ONE account, ever.** The *first* account you make during account registration is your one and only osu! account, from the moment of its creation and forevermore. This account is tied to YOUR BODY. It is not anyone else's — not your brother's, your mother's, your sister's, your friend's — it is YOURS. Don't share your account between multiple bodies. You don't get to make a new one if you lose access to it. Keep it safe.

**Comment by TicClick** [comment]
I am going to go anead and close this one, sorry. we can't and won't account for all possible corner cases in favour of odd wordings which will only bring more confusion and discord. if any of you are in doubt, I am sure you 1) can apply common sense and read the spirit of the rule (one physical being = one account) instead of trying to extrapolate, and 2) can contact the support to find out more about your exact situation.

tl;dr: overly specific, unnecessary
